<?xml version="1.0" encoding="utf-8"?>
<kenticocommunity.qandaanswerdata>
  <QAndAAnswerDataAuthorMemberID>
    <GUID>8fb0dad6-8c28-43e4-849b-684df65417a7</GUID>
    <ObjectType>cms.member</ObjectType>
  </QAndAAnswerDataAuthorMemberID>
  <QAndAAnswerDataCodeName>Hi-Tim-You-are-not-alone-it-is-been-3-dayse396e20c</QAndAAnswerDataCodeName>
  <QAndAAnswerDataContent>
<![CDATA[
Hi Tim,

You are not alone, it is been 3 days i had thr same issue i looked on mostly all attach nuggets and i still not find the issue. But when i updated dancing goat it worked. Community portal gave the same issue also after version 30.9.0
]]>
  </QAndAAnswerDataContent>
  <QAndAAnswerDataDateCreated>2025-08-30 19:42:44Z</QAndAAnswerDataDateCreated>
  <QAndAAnswerDataGUID>b65afdf7-bb67-4e90-bbcf-241d39d2a8e5</QAndAAnswerDataGUID>
  <QAndAAnswerDataQuestionWebPageItemID>
    <CodeName>Getting-exception-after-upgrading-to-30-9-1-a8037911-wkph1uu7</CodeName>
    <GUID>074231a9-7fc8-4522-a91e-af5039389255</GUID>
    <ObjectType>cms.webpageitem</ObjectType>
  </QAndAAnswerDataQuestionWebPageItemID>
  <QAndAAnswerDataWebsiteChannelID>
    <GUID>9ca57e0f-969f-43be-91d3-6c45659316bd</GUID>
    <ObjectType>cms.websitechannel</ObjectType>
    <Parent>
      <CodeName>devnet</CodeName>
      <GUID>4f636110-fd4a-4905-83e6-998752c2b2c2</GUID>
      <ObjectType>cms.channel</ObjectType>
    </Parent>
  </QAndAAnswerDataWebsiteChannelID>
</kenticocommunity.qandaanswerdata>